package/gluon-mesh-vpn-tunneldigger/luasrc/lib/lua/gluon/mesh-vpn/method/tunneldigger.lua

```diff
@@ -0,0 +1,33 @@
+local uci = require("simple-uci").cursor()
+
+local site = require 'gluon.site'
+
+local M = {}
+
+function M.public_key()
+	return nil
+end
+
+function M.enable(val)
+	uci:set("tunneldigger", "mesh_vpn", "enabled", val)
+	uci:save("tunneldigger")
+end
+
+function M.active()
```

Comment:
As there are multiple vpn methods, that are available but only one that is returned using `M.get_active_method()` this function might better be called `available()` to avoid a false affordance.
```suggestion
function M.available()
```

Reply:
As explained to @lemoer , this is intended to have different methods for different domains but one vpn method active per domain. SO I'd stick with active here.